ssl-config-core/src/main/scala/com/typesafe/sslconfig/ssl/SelfSigned.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2015 - 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.typesafe.sslconfig.ssl
+
+import java.security.KeyStore
+
+import javax.net.ssl._
+
+/**
+ * Contains a statically initialized self-signed certificate.
+ *
+ * Was: play.it.test.ServerEndpoint.SelfSigned
+ */
+object SelfSigned {
+  /**
+   * The SSLContext and TrustManager associated with the self-signed certificate.
+   */
+  lazy val (sslContext, trustManager): (SSLContext, X509TrustManager) = {
```

Comment:
Actually, this should be in a dedicated "testkit"-like module (a new one) of ssl-config.

Reply:
Looks like a minor thing to do. Do you want to solve this at this PR or in a new one?